utils/secrets.ts

```diff
@@ -6,4 +6,5 @@ export const STORAGE_BUCKET = process.env.NEXT_PUBLIC_STORAGE_BUCKET;
 export const MESSAGING_SENDER_ID = process.env.NEXT_PUBLIC_MESSAGING_SENDER_ID;
 export const APP_ID = process.env.NEXT_PUBLIC_APP_ID;
 export const MEASUREMENT_ID = process.env.NEXT_PUBLIC_MEASUREMENT_ID;
-export const BE_DEPLOYMENT_DOMAIN = process.env.NEXT_PUBLIC_BE_DEPLOYMENT_DOMAIN;
\ No newline at end of file
+export const BE_DEPLOYMENT_DOMAIN =
```

Comment:
Why do we need this change? Can we just remove changes to this file?

Reply:
Yep, will do